docs/Getting-started.md

```diff
@@ -0,0 +1,52 @@
+# Start contributing to the Performance plugin
+This guide is intended to give details on how to contribute to the Performance plugin. From setting up the development environment to creating pull requests.
+
+## Prerequisites
+- Node.js
+- Docker
+- [Git](https://git-scm.com)
+
+## Setup the development environment
+
+### Clone the repository
+Start by cloning the repository into your local machine. To do this, run the following command in your terminal:
+`git clone https://github.com/WordPress/performance.git`
```

Comment:
I think for a better reading flow, it would help to put all full commands in their own line, wrapping it with ``` instead of `. Comparing https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/getting-started-with-code-contribution.md with https://github.com/WordPress/performance/blob/add/39-contributing-documentation/docs/Getting-started.md, I think the Gutenberg doc is easier to follow with its commands broken out in individual lines.

If you agree, let's apply the change to this whole doc?

Reply:
I'm not sure if I correctly understand this @felixarntz. Can you add a snippet to illustrate?

Reply:
@JustinyAhin I mean like this:
```
git clone https://github.com/WordPress/performance.git
```

If you put ``` around code and split it between lines, it renders in its own code block rather than inline, which makes it easier to spot.

Reply:
Oh you are right, this is much cleaner.